UI/window-basic-status-bar.cpp

```diff
@@ -195,7 +195,11 @@ void OBSBasicStatusBar::UpdateBandwidth()
 	if (++seconds < bitrateUpdateSeconds)
 		return;
 
-	uint64_t bytesSent = obs_output_get_total_bytes(streamOutput);
+	OBSOutputAutoRelease output = obs_weak_output_get_output(streamOutput);
+	if (!output)
+		return;
```

Comment:
Just to confirm (can’t check atm), there are no values displayed that should potentially get reset / set to zero when the output is gone?

Reply:
When the stream is stopped `StreamStopped` is called which:
1. calls `ReconnectClear()`
   1. resets the various time/bytes members
2. sets `streamOutput` to `nullptr`
3. calls `Deactivate()`
   1. resets the UI elements in the status bar

Aside from that, I'm not sure what happens if the output disappears otherwise.

Reply:
As far as I can tell from off-thread discussion, this concern was resolved.